Tools/Scripts/run-webdriver-tests

```diff
@@ -53,8 +53,12 @@ option_parser.add_option('--display-server', choices=['xvfb', 'xorg', 'weston',
                          help='"xvfb": Use a virtualized X11 server. "xorg": Use the current X11 session. '
                               '"weston": Use a virtualized Weston server. "wayland": Use the current wayland session.'
                               '"headless": Headless mode in current session')
-option_parser.add_option('--enable-webdriver-bidi', action='store_true',
-                         help='Run tests with BiDi enabled')
+option_parser.add_option('--enable-webdriver-bidi', action='store_const',
+                         default=True, const=True, dest='enable_webdriver_bidi',
+                         help='Run Selenium tests with BiDi enabled')
+option_parser.add_option('--disable-webdriver-bidi', action='store_const',
+                         const=False, dest='enable_webdriver_bidi',
+                         help='Run Selenium tests with BiDi disabled')
```

Comment:
The `--enable-webdriver-bidi` switch looks a bit redundant and can be confusing regarding to what one is the current default. I would just remove it and keep only the `--disable-webdriver-bidi` one.

But is fine in any case.